src/components/UnstyledLink/UnstyledLink.tsx

```diff
@@ -23,4 +23,4 @@ function UnstyledLink(props: Props) {
   );
 }
 
-export default React.memo(UnstyledLink);
```

Comment:
Unfortunately  `enzyme` is having trouble `.find(UnstyledLink)`. We're not going to have a noticeable performance difference from this so I'm removing it for now till we find the root cause/enzyme fixes it.